plugins/parsers/bindata/README.md

```diff
@@ -0,0 +1,55 @@
+# BinData
+
+The "BinData" parser translates binary records consisting of multiple fields into Telegraf metrics. It supports:
+
+- Little- and Big-Endian encoding
+- bool, int8/uint8, int16/uint16, int32/uint32, int64/uint64, float32/float64 field types
+- UTF-8 and ASCII-encoded strings
+- unix, unix_ms, unix_us and unix_ns timestamp
+
+### Configuration
+
+```toml
+[[inputs.mqtt_consumer]]
+  name_override = "drone_status"
+
+  ## Data format to consume.
+  ## Each data format has its own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
+  data_format = "bindata"
+
+  ## Numeric fields endiannes, "be" or "le", default "be"
+  # bindata_endiannes = "be"
```

Comment:
I think the default should be `host`, i.e. do not mess with endianess?

Reply:
@srebhan Could you please elaborate on this? Not sure whether I understand this comment.

Reply:
What I mean is that people tend to not knowing the endianess of data and that's fine as long as the do not cross endianess boarders. If you for example generate the data in a machine (with its endianess) and the parse it on that same machine, you probably do not need to know if that host has big- or little-endian. Therefore I suggest a shortcut for these use-cases like using `native` or `host` as endianess instead of explicitly defining it. Does that make things clearer?